Core/Core/Use Cases/GetAssignments.swift

```diff
@@ -100,35 +98,3 @@ public class GetSubmittableAssignments: GetAssignments {
         return Scope(predicate: predicate, order: [ a, b ])
     }
 }
-
-public class GetSyllabusAssignments: GetAssignments {
```

Comment:
Was this not being used anywhere?

Reply:
Nope. Syllabus uses `/calendar_events?type=assignment` now instead. I noticed this when I went looking for `clearsOnWrite`